packages/aws-cdk-lib/aws-stepfunctions-tasks/test/bedrock/invoke-model.test.ts

```diff
@@ -234,8 +234,6 @@ describe('Invoke Model', () => {
         ],
       },
       End: true,
-      InputPath: '$.prompt',
-      OutputPath: '$.prompt',
```

Comment:
Are there other unit tests that test this functionality?

Reply:
There are two unit tests, one to check the expected template and one to check the permissions populated in policy

Reply:
The title of this unit test is "invoke model allows input and output json path". I think this test was originally meant to test the original behavior of inputPath and outputPath. So we need an additional new test for these new s3InputUri and s3OutputUri params, AND fix this unit test to go back to testing the original behavior of inputPath and outputPath.

Reply:
Update: I see in the history now that you had actually added this test in the last change. I do still think we should have both tests (inputPath and s3InputUri)

Reply:
Thanks @clareliguori , this leads to one question can `body` be skipped in case of an `inputPath` ? since the earlier validation seems to added only for `body` and `input` and didn't point to a definition where we might be using these base props.